plugin/API/APIAvailabilityChecker.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace onOffice\WPlugin\API;
+
+
+use onOffice\SDK\onOfficeSDK;
+use onOffice\WPlugin\SDKWrapper;
+
+class APIAvailabilityChecker
+{
+	/** @var SDKWrapper */
+	private $_pSDKWrapper = null;
+
+	/**
+	 *
+	 * @param SDKWrapper $pSDKWrapper
+	 */
+
+	public function __construct(SDKWrapper $pSDKWrapper = null)
```

Comment:
` = null` and `?? new SDKWrapper` aren't necessary, the dependency gets resolved using autowiring